src/styles/bloks/calendar__day.scss

```diff
@@ -0,0 +1,45 @@
+.calendar__day {
+  @include square ($ceil-size);
+  border: 1px solid black;
+  background-color: #eee;
+  border-radius: 10px;
+  cursor: pointer;
+  transition: background-color 0.5s, transform 0.5s;
+
+  &:hover {
+    transform: translateY(-20px);
+    background-color: #f5c2cb;
+  }
+
+  @for $index from 1 through 31 {
+    &:nth-child(#{$index})::before {
+      content: "#{$index}";
+      display: flex;
+      justify-content: center;
+      line-height: $ceil-size;
+      font-family: Arial, Helvetica, sans-serif;
+      font-size: 30px;
+      color: black;
```

Comment:
no need to set all styles for each element separately

just set common styles to `calendar__day`